ci/deps/azure-36-locale_slow.yaml

```diff
@@ -3,8 +3,16 @@ channels:
   - defaults
   - conda-forge
 dependencies:
+  - python=3.6.*
+  # tools
+  - cython=0.29.13
+  - pytest
```

Comment:
need a min pin on pytest (for all .yaml)

Reply:
We've got all these versions in the builds:
- pytest
- pytest>=4.0.2
- pytest>=4.0.2,<5.0
- pytest=4.5.0
- pytest>=5.0.0
- pytest>=5.0.1

Any preference on which you want to require? `pytest>-5.0.1` is ok?

Reply:
yeah ok with the >=4.0.2 I think < 4 we don't support at all